test/StreamManager.test.js

```diff
@@ -166,6 +171,52 @@ describe("StreamManager", function () {
     .to.be.revertedWith('NotPayer');
   })
 
+  // Returning 0, because the current timestamp is less than the sum of the thread creation time and the "cliff" period 
```

Comment:
```suggestion
  // Returning 0, because the current timestamp is less than the sum of the stream creation time and the "cliff" period 
```

Reply:
@CTAPCKPIM I think it's just what you wrote.

Reply:
Fixed